Sources/Amplitude/Plugins/iOS/UIKitElementInteractions.swift

```diff
@@ -115,6 +123,13 @@ extension UIGestureRecognizer {
 
         guard state == .ended, let view else { return }
 
+        // TODO: Reduce SwiftUI noise by filtering out system gesture recognizers.
+        // Currently, the gesture recognizers with a SwiftUI view are blocked.
+        let viewClass = NSStringFromClass(type(of: view as AnyObject))
+        if viewClass.contains("SwiftUI") {
+            return
+        }
+
```

Comment:
I think I just have to check if the view is a known type that we want to block instead of just checking the name for "SwiftUI". I will update this shortly.